eo-maven-plugin/src/main/java/org/eolang/maven/DepDirs.java

```diff
@@ -60,7 +60,7 @@ final class DepDirs extends ListEnvelope<String> {
      */
     private static List<String> list(final Path dir) throws IOException {
         final List<String> names = new LinkedList<>();
-        if (new Home().exists(dir)) {
+        if (Files.exists(Home.clean(dir))) {
```

Comment:
as I understand, it's a hack to make current code to work and make tests to pass. It's fine. But after all it's better to replace all `Files.exists()` invocations to `Home.exists()`. I don't think we need to use something outside of the `Home` folder. In other words, we need a puzzle here.

Reply:
@volodya-lombrozo In this case `DepDirs` just want to check existence of an absolute path. I think it's irrelevant to `Home` object (with current design of `DepDirs`). However we still want to clean provided path.
We may create a puzzle here though to refactor `DepDirs` to use `Home` object.

Reply:
@volodya-lombrozo Actually to use `Home` object `DepDirs` has to change its concept. Currently it encapsulate a path. There is nothing about any base folder and its subfolders. I would say we don't need `Home` here at all.

Reply:
Ok